projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -688,6 +688,9 @@ public static function enqueue_block_editor_assets() {
 			$is_current_user_connected = ( new Connection_Manager( 'jetpack' ) )->is_user_connected();
 		}
 
+		if ( self::blocks_variation() === 'experimental' && $is_current_user_connected ) {
+			wp_enqueue_style( 'recoleta-font', '//s1.wp.com/i/fonts/recoleta/css/400.min.css', array(), Constants::get_constant( 'JETPACK__VERSION' ) );
+		}
```

Comment:
@saygunnyc both @jeherve  and @monsieur-z make this point. About not using the Recoleta font. 
Should we try to update the modal to not use it? The modal would look something like this. 
<img width="765" alt="Screenshot 2024-01-12 at 8 57 43 AM" src="https://github.com/Automattic/jetpack/assets/115071/dbf731dd-14aa-4121-9e54-f5229287ac6c">

Reply:
I chatted with @saygunnyc about this and ideally all the modals should be more aligned. Most of the Jetpack Mobile App Notices have this font on .com. 
I am not sure when we are going to revisit some of the other modals in Gutenberg but at this point lets ship this. 
Any reason not to ship it? Besides the extra bytes (34kb). Should we include a filter so folks can remove this feature in Jetpack if they are not using it? 
From testing in FF the font only gets downloaded once the font is used. (Modal is opened)

Reply:
If this is meant to align the design with other existing notices on WordPress.com, do we need to add the font here? I expect it would already be added somewhere else on WordPress.com to handle the design of those other notices. Maybe we can rely on that, and fallback to a font family used on Core when that font is not available, to handle the design in core?

Reply:
> If this is meant to align the design with other existing notices on WordPress.com, do we need to add the font here?
Yes. Other wise the font is not available in the browser. 

I don't really want to add more complexity to the code and have 2 different UIs just So we don't download the font in the browser for Jetpack users. The browser is already smart enough to figure out if it needs to download the font. At least for me the font only downloads when the modal is opened. See https://jaketrent.com/post/when-font-face-is-downloaded/ 

Or are there other concerns? 

> Aligning with core design elements. 
The UI is as much of a ty into to Jetpack app (marketing) as it is a UI element. 
For example in WordPress.com/discover we use a similar element like this. 
<img width="1096" alt="Screenshot 2024-01-18 at 11 02 36 AM" src="https://github.com/Automattic/jetpack/assets/115071/e677df6a-8e5f-4b99-aca3-f3b74b8e8838">

The font that tells you about why to get the app is the same.

Reply:
> Yes. Other wise the font is not available in the browser.

Right, that means that there is currently no other element in the block editor that uses that font. Adding the font now for this one modal will break consistency in the design.

I'm no designer though, so I'd appreciate if someone from a design team would chime in on that :)